app/src/main/java/org/oppia/app/player/state/itemviewmodel/SelectionInteractionViewModel.kt

```diff
@@ -7,12 +7,16 @@ import org.oppia.app.model.Interaction
 import org.oppia.app.model.InteractionObject
 import org.oppia.app.model.StringList
 import org.oppia.app.model.UserAnswer
-import org.oppia.app.player.state.SelectionItemInputType
 import org.oppia.app.player.state.answerhandling.InteractionAnswerErrorOrAvailabilityCheckReceiver
 import org.oppia.app.player.state.answerhandling.InteractionAnswerHandler
 import org.oppia.app.player.state.answerhandling.InteractionAnswerReceiver
 import org.oppia.app.viewmodel.ObservableArrayList
 
+enum class SelectionItemInputType {
+  CHECKBOXES,
+  RADIO_BUTTONS
+}
+
```

Comment:
Why is this needed?

Reply:
This view model was depending on `SelectionInteractionView` only because it housed this enum class while `SelectionInteractionView` was depending on `SelectionInteractionViewModel` in createAdapter(). So, I decided to move the enum class to the view model and only have `SelectionInteractionView` depend on `SelectionInteractionViewModel`.

Reply:
Thanks for clarifying. Please keep the original KDoc if this is the new place for the enum to exist.

Reply:
Okay!